e2e/remotetasks/remotetasks.go

```diff
@@ -4,14 +4,15 @@
 package remotetasks
```

Comment:
Should we just remove these tests and the related ECS infra? Let's see what @schmichael thinks. (But probably in a separate PR anyways.)

Reply:
Please remove. We removed RTD from the docs and archived the repo. Not a bad idea, but incomplete and lacking in demand.